app/modules/ia_config_reader.py

```diff
@@ -85,7 +85,14 @@ def get_identifiers_list(self, genus_species, ia_class):
     def get_identifiers_dict(self, genus_species, ia_class):
         identifiers = self.get_identifiers_with_links(genus_species, ia_class)
         identifiers_dict = self._resolve_links_to_dict(identifiers)
-        return identifiers_dict
+        # trim the '_identifiers.' prefix off the keys
```

Comment:
I'm not clear why you changed this. It doesn't seem related to the other changes in the code. WIP?

Reply:
just making note what we covered in slack:  this is basically a fix for a bug in the parsing of ia_config data, in order to give frontend the desired json values.  (not truly related to this PR other than it has to do with identification config.)